packages/hash/integration/src/tests/model/knowledge/orgMembership.model.test.ts

```diff
@@ -0,0 +1,96 @@
+import { getRequiredEnv } from "@hashintel/hash-backend-utils/environment";
+import { createGraphClient } from "@hashintel/hash-api/src/graph";
+import {
+  ensureWorkspaceTypesExist,
+  WORKSPACE_TYPES,
+} from "@hashintel/hash-api/src/graph/workspace-types";
+import {
+  OrgMembershipModel,
+  OrgModel,
+  OrgSize,
+  UserModel,
+} from "@hashintel/hash-api/src/model";
+import { Logger } from "@hashintel/hash-backend-utils/logger";
+import {
+  createKratosIdentity,
+  adminKratosSdk,
+} from "@hashintel/hash-api/src/auth/ory-kratos";
+
+jest.setTimeout(60000);
+
+const logger = new Logger({
+  mode: "dev",
+  level: "debug",
+  serviceName: "org-membership-tests",
+});
+
+const graphApiHost = getRequiredEnv("HASH_GRAPH_API_HOST");
+const graphApiPort = parseInt(getRequiredEnv("HASH_GRAPH_API_PORT"), 10);
+
+const graphApi = createGraphClient(logger, {
+  host: graphApiHost,
+  port: graphApiPort,
+});
+
+describe("OrgMembership model class", () => {
+  let kratosIdentityId: string;
+
+  let testUser: UserModel;
+
+  let testOrg: OrgModel;
+
+  beforeAll(async () => {
+    await ensureWorkspaceTypesExist({ graphApi, logger });
+
+    const testUserIdentity = await createKratosIdentity({
+      traits: {
+        emails: ["test@example.com"],
+      },
+    });
+
+    kratosIdentityId = testUserIdentity.id;
+
+    testUser = await UserModel.createUser(graphApi, {
+      emails: ["alice@example.com"],
+      kratosIdentityId,
+    });
+
+    testOrg = await OrgModel.createOrg(graphApi, {
+      name: "Test org",
+      shortname: "test-org",
+      providedInfo: {
+        orgSize: OrgSize.ElevenToFifty,
+      },
+    });
+  });
+
+  let testOrgMembership: OrgMembershipModel;
+
+  it("can create an OrgMembership", async () => {
+    testOrgMembership = await OrgMembershipModel.createOrgMembership(graphApi, {
+      responsibility: "test",
+      org: testOrg,
+    });
+
+    await testUser.createOutgoingLink(graphApi, {
+      linkTypeModel: WORKSPACE_TYPES.linkType.hasMembership,
+      targetEntityModel: testOrgMembership,
+    });
+  });
+
+  it("can get the org of an org membership", async () => {
+    const fetchedOrg = await testOrgMembership.getOrg(graphApi);
+
+    expect(fetchedOrg).toEqual(testOrg);
+  });
+
+  it("can get the user of an org membership", async () => {
+    const fetchedUser = await testOrgMembership.getUser(graphApi);
+
+    expect(fetchedUser?.entityId).toEqual(testUser.entityId);
+  });
+
+  afterAll(async () => {
+    await adminKratosSdk.adminDeleteIdentity(kratosIdentityId);
```

Comment:
If this is worth doing then we should port this over to the other model classes because the kratos Identities are not cleaned up

Reply:
I've remove this as https://github.com/hashintel/hash/pull/1068/commits/3823148f0cdfc50b69f2cd9ef08ba1f21fa49635 means it's no longer neccessary (as the `email` associated with the kratos identity is unique for each test-run)

Reply:
Realised we still need this for the user model tests because of how we create the identity in that file. Have added it back in https://github.com/hashintel/hash/pull/1068/commits/12b3600a976db8ac214aa66f8934d3317fbe51d4, but kept the removal from the org model tests where we use the `createTestUser` method.